src/js/components/RangeInput/StyledRangeInput.js

```diff
@@ -138,6 +139,16 @@ const trackColorStyle = (props) => {
   `;
 };
 
+const disabledRangeInputStyle = (props, context) => css`
+  ${disabledStyle(props.theme.rangeInput.disabled.opacity)}
+  ${props.theme.rangeInput.disabled[context] &&
+  props.theme.rangeInput.disabled[context].color &&
```

Comment:
```suggestion
```
Try the two above code suggestions locally, but just trying the "?" notation as a way to simplify slightly here also.

Reply:
sounds good Ill try it locally